src/sql/base/browser/ui/infoButton/infoButton.ts

```diff
@@ -75,6 +83,11 @@ export class InfoButton extends sqlButton {
 		this.element.style.background = 'none';
 
 		this.infoButtonOptions = options;
+		this._register(addDisposableListener(this.element, EventType.MOUSE_OVER, e => {
```

Comment:
What about mouse out?

Typically for hover states you'd use the hover CSS selector for this anyways instead of manually hooking it up. See https://github.com/microsoft/azuredatastudio/blob/c39c20cd4ba72d432cc37c307f158e2a22e3861b/src/sql/base/browser/ui/panel/panel.ts#L334 for an example of how this is hooked up during the style call

Reply:
both are mouse in and out are being handled in the base class, I removed this and fully leverage the base class.